concourse/Makefile

```
@@ -240,28 +240,25 @@ set-cloudbuild-pipeline-base:
 		--pipeline $(PIPELINE_NAME) \
 		${FLY_OPTION_NON-INTERACTIVE} || echo "pipelines/cloudbuild_pipeline.yml has errors"
 
-.PHONY: pivnet
-pivnet: set-pivnet-pipeline
-
-.PHONY: set-pivnet-pipeline
-set-pivnet-pipeline: generate-pivnet-pipeline
+.PHONY: set-gcs-pipeline
+set-gcs-pipeline: generate-gcs-pipeline
```

this new line should probably go in the common section of short goals on line 64.
```
gpdb-artifacts: set-gpdb-artifacts-pipeline
```

```suggestion
set-gpdb-artifacts-pipeline: generate-gpdb-artifacts-pipeline
```